src/components/v5/shared/Stepper/partials/StepperButton/StepperButton.tsx

```diff
@@ -21,18 +21,27 @@ const StepperButton: React.FC<StepperButtonProps> = ({
   tooltipProps,
   ...rest
 }) => {
+  const ref = useRef<HTMLButtonElement>(null);
   const Icon = icon || ICON_NAME_MAP[stage];
   const isMobile = useMobile();
 
+  useEffect(() => {
+    if (ref?.current && isHighlighted) {
+      ref.current.scrollIntoView({ behavior: 'smooth', inline: 'end' });
```

Comment:
Great start on your 1st PR @mmioana ! 🥳 

Code-wise all good! This is not a change request but thought I'd say this would be pretty cool as a hook i.e.
```js
const { ref, scrollIntoView } = useScrollIntoView(withWhateverParams)
```